src/app/_components/SettingsAlert.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+
+type SettingsAlertProps = {
+  showAlert: boolean;
+};
+
+export default function SettingsAlert({ showAlert }: SettingsAlertProps) {
+  const [isVisible, setIsVisible] = useState(false);
+
+  useEffect(() => {
+    if (showAlert) {
+      setIsVisible(true);
+      // Automatically hide the message after 3 seconds (3000 milliseconds)
+      const timeout = setTimeout(() => {
+        setIsVisible(false);
+      }, 3000);
+      return () => clearTimeout(timeout);
+    }
+  }, [showAlert]);
+
+  return (
+    <div
+      role="alert"
+      className={`alert bg-white shadow-md transition duration-1000 ease-in-out ${
+        isVisible ? "opacity-100" : "opacity-0"
```

Comment:
tailwind has a visibility utility class for this instead of toggling opacity - [link to docs](https://v1.tailwindcss.com/docs/visibility)

react also has a hidden property - [link to stackoverflow](https://stackoverflow.com/questions/45608085/placing-hidden-attribute-with-react-in-tag)

Reply:
scratch that, i didn't notice the opacity was for the transition animation.